### ISSUE TITLE
bad argument #2 to 'blend' (no value)

### ISSUE THREAD

**Issue by danielnehrig** [rcarriga/nvim-notify#20]
Error: `EError running notification service: ...opt/nvim-notify/lua/notify/service/buffer/highlights.lua:59: bad argument #2 to 'blend' (no value)`

the highlight used for notify because normal is none for me
```lua
cmd("autocmd ColorScheme * highlight NotifyBG guibg=#3d3d3d guifg=#3e4451")
```

code in context
```lua
function Make:Report(msg)
    vim.cmd [[packadd nvim-notify]]
    local opt = {
        title = "Neomake"
    }
    local info = vim.g.neomake_hook_context.jobinfo
    local notify = require("notify")
    notify.setup(
        {
            -- Animation style (see below for details)
            stages = "fade",
            -- Default timeout for notifications
            timeout = 3000,
            -- For stages that change opacity this is treated as the highlight behind the window
            background_colour = "NotifyBG",
            -- Icons for the different levels
            icons = {
                ERROR = "",
                WARN = "",
                INFO = "",
                DEBUG = "",
                TRACE = "✎"
            }
        }
    )
    if info.exit_code == 0 then
        notify("Job Finished Successfully", _, opt)
    elseif info.exit_code == 1 then
        notify("Job Failed", "error", opt)
    else
        notify("Job Started", "log", opt)
    end
end
```

**Comment by rcarriga** [comment]
This should be solved by #19, please try the latest commit

**Comment by danielnehrig** [comment]
can confirm works now thanks
